dotcom-rendering/src/web/components/ArticleMeta.tsx

```diff
@@ -23,6 +23,7 @@ type Props = {
 	primaryDateline: string;
 	secondaryDateline: string;
 	branding?: Branding;
+	hasByline?: boolean;
```

Comment:
can this line be removed?

Reply:
Nevermind seems like github has just bugged out!

Reply:
This is part of the temporary fix.